webcomponents/search-webcomponent/src/app/shared/form/form-field-control.service.ts

```diff
@@ -1,20 +1,74 @@
 import {Injectable} from '@angular/core';
 import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {ValidationMessages} from '../../utils/ValidationMessages';
 
 import {FormFieldBase} from './form-field-base';
 
 @Injectable()
 export class FormFieldControlService {
   constructor() {}
 
+  private emailPatternRegex: string =
+    '^[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,4}$';
+  private zipCodePatternRegex: string = '[0-9]{4}';
+  private numberPatternRegex: string = '[0-9]';
```

Comment:
Is this correct? Only a single digit for number fields would mean we cant input numbers for the price or anything above 9.

Reply:
You are right that's wrong, changed it to '^((?![0-9]).)*$'

Reply:
Wouldn't it be more something like `^([0-9])+$` ? The one you wrote matches any character *except* for numbers between 0 and infinite times

Reply:
Okay I think you coded the right thing, just the one in the comment isn't the right one

Reply:
:S yea sorry was testing the other as well and copy pasted wrong...